components/rt/src/rt_estimator.m

```diff
@@ -205,7 +205,7 @@
 
 end
 
-disp(' > Simulation done.')
+%disp(' > Simulation done.')
```

Comment:
All the `disp` commands are added for clarity when running the pipeline. Why remove them?

Reply:
Why shoulld we keep displaying things if the final user would only get the file? Unless we want to keep them for us during the development phase

Reply:
these `disp` are printed to the log files, and are essential for swift debugging. The more verbose the script is, the easier it is to debug later on. They will not be displayed to the web, but are logged only.

Reply:
I see, so I'll add some `disp` to reff as welll